client/InvoiceTickets.js

```diff
@@ -10,10 +22,38 @@ Template.InvoiceTickets.helpers({
     return InvoiceTickets.byTimeRange(
       FlowRouter.getParam("filter"),
       FlowRouter.getQueryParam("sortBy"),
-      FlowRouter.getQueryParam("sortOrder")
+      FlowRouter.getQueryParam("sortOrder"),
+      Session.get("itemsLimit")
     );
   },
   formatDate: function(date){
     return moment(date).format("MM-DD-YYYY");
   },
 });
+Template.InvoiceTickets.moreResults = function() {
+    // If, once the subscription is ready, we have less rows than we
+    // asked for, we've got all the rows in the collection.
+    return !(InvoiceTickets.find().count() < Session.get("itemsLimit"));
```

Comment:
Ok but at some point (at the end of the list) you might run into the trouble that your `itemsLimit` is higher than the actual total number of invoices and thus your "loading" indicator will be visible although there is nothing more to be loaded (i haven't tried it out, just assuming here)